static/oas/v1/sports/components/rosters.yml

```diff
@@ -63,18 +63,40 @@ components:
                 type: array
                 items: 
                   $ref: "../../users/users.yml#/schemas/User"
-    PostRosterResponse:
+                            
+    PostRosterRequest:
       type: object
       properties:
-        status_code:
-          $ref: "../../common/schemas.yml#/components/schemas/StatusCode200"
-        status_string:
-          $ref: "../../common/schemas.yml#/components/schemas/StatusString200"
-        request_id:
-          $ref: "../../common/schemas.yml#/components/schemas/RequestId"
-    PostRosterRequest:
+        captain_email:
```

Comment:
You'll need to pull latest as this has been updated to IDs. Also would be great if we could again re-use RosterMOdel but omit the ID.